custom_components/healthchecksio/binary_sensor.py

```diff
@@ -46,7 +46,7 @@ async def async_update(self):
             if self.unique_id == check.get("ping_url").split("/")[-1]:
                 self.check = check
                 break
-        self._status = self.check.get("status") == "up"
+        self._status = self.check.get("status") in ["up", "grace"] 
```

Comment:
```suggestion
        self._status = self.check.get("status") != "down"
```

As there are many more, and this will be breaking anyway, its better to switch it around.
> The possible values for the status field are: new, started, up, grace, down, and paused.